player/command.c

```diff
@@ -2481,6 +2482,18 @@ static int property_imgparams(const struct mp_image_params *p, int action, void
         {0}
     };
 
+    if (action == M_PROPERTY_KEY_ACTION) {
```

Comment:
What happens if you set aspect at different levels of video-*-params?

Reply:
A set is exactly the same. It writes to `video-aspect-override` either way. The difference would be if you use `add` or `multiply` on it. The what the amount gets added/multiplied to depends on whatever the aspect of the specific video-*-params happens to be.